languages/ruby/bitwarden_sdk/lib/projects.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require_relative 'bitwarden_error'
+
+module BitwardenSDK
+  class ProjectsClient
+    def initialize(command_runner)
+      @command_runner = command_runner
+    end
+
+    def create_project(project_name, organization_id)
+      project_create_request = ProjectCreateRequest.new(
+        project_create_request_name: project_name,
+        organization_id: organization_id
+      )
+      command = create_command(
+        create: project_create_request
+      )
+      response = parse_response(command)
+
+      projects_response = ResponseForProjectResponse.from_json!(response).to_dynamic
+
+      raise BitwardenError, projects_response['errorMessage'] if projects_response['errorMessage']
```

Comment:
We have a `Success` flag on the response models, while I don't believe it should happen, it would technically be possible for a failure to not have an error message.

Reply:
it is refactored all over clients that we check existence of keys before getting values